build/entrypoint-tools.sh

```diff
@@ -3,9 +3,14 @@
 echo "Starting tools container ${RAILS_ENV}"
 
 # Get setup to be able to do stuff
+echo "Installing gems..."
 ./build/install_gems.sh
 
+# Start init
+echo "Starting init"
+exec /sbin/init
+
 # Sleep forever, sweet container
-while `true`; do
-   sleep 600
-done
+#while `true`; do
```

Comment:
intentional, yeah? No more safety fall through

Reply:
yes, instead of a `sleep` loop, it's running `exec /sbin/init` so `init` will become the primary process